api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -226,8 +226,9 @@ public List<ReportingUser> getUsers(long limit, long offset) {
                 + "  u.disabled,\n"
                 + "  uame.era_commons_bypass_time,\n"
                 + "  uame.era_commons_completion_time,\n"
-                + "  uaml.ras_login_gov_bypass_time,\n"
-                + "  uaml.ras_login_gov_completion_time,\n"
+                + "  uami.identity_bypass_time,\n"
```

Comment:
I don't think this is going to work - where is this value set?

Reply:
Good catch! Thank you! I fixed it now. They're set on lines 322 and 323.